### PR TITLE
feat: add autocompletion for azioncli

### PR DESCRIPTION
**Pull request by srmaxmazur**
WHAT
Added autocomplete for commands and subcommands.

### CODE REVIEW

File: .goreleaser.yaml [hunk 1]
```diff
@@ -1,31 +1,46 @@
...
+    - chmod +x ./scripts/completions.sh
+    - ./scripts/completions.sh
 
 brews:
-- name: azioncli
-  homepage: https://github.com/aziontech/azion-cli
-  description: The developer friendly way to interact with Azion
-  license: "MIT"
-  dependencies:
+  - name: azioncli
```
Comment (PatrickMenoti) [inline .goreleaser.yaml:30]: same as above... why change the identation here? did the previous one generate an error for you?

**Comment by srmaxmazur** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by MaxwelMazur** [comment]
I have read the CLA Document and I hereby sign the CLA

File: README.md [hunk 2]
```diff
@@ -43,6 +43,32 @@ $ make build ENVFILE=./env/prod
 $ make cross-build
 ```
 
+## Setup Auto Complete
+> Tip for each step restart the terminal
```
Comment (PatrickMenoti) [inline README.md:47]: Tip for each step: restart your terminal

File: .goreleaser.yaml (hunk 1 above)
Comment (srmaxmazur) [reply .goreleaser.yaml:30]: I don't know how to answer you, I don't know how this field got there, in my place I ended up not seeing it and it doesn't reflect the same

File: .goreleaser.yaml [hunk 3]
```diff
@@ -1,31 +1,46 @@
...
+
 builds:
   - env:
       - CGO_ENABLED=0
     main: ./cmd/azioncli
     ldflags:
-        - -X github.com/aziontech/azion-cli/pkg/cmd/version.BinVersion={{ .Env.BIN_VERSION }}
-        - -X github.com/aziontech/azion-cli/pkg/constants.AuthURL={{ .Env.AUTH_URL }}
-        - -X github.com/aziontech/azion-cli/pkg/constants.ApiURL={{ .Env.API_URL }}
+      - -X github.com/aziontech/azion-cli/pkg/cmd/version.BinVersion={{ .Env.BIN_VERSION }}
```
[the remark this answers is not in the document]
Comment (srmaxmazur) [reply .goreleaser.yaml:8]: I don't know how to answer you, I don't know how this field got there, in my place I ended up not seeing it and it doesn't reflect the same

[the remark this answers is not in the document]
Comment (srmaxmazur) [reply .goreleaser.yaml:8]: what i added - binary: azioncli

**Review by PatrickMenoti — APPROVED** [review]
LGTM

**Review by pauloSF0 — APPROVED** [review]

**Pull request merged**